src/main/java/com/mongodb/jdbc/oidc/RFC252ServerMain.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2024-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.jdbc.oidc;
+
+import java.io.IOException;
+/**
+ * Main class to start the RFC8252HttpServer and wait for the OIDC response Used for testing the
+ * serving of the HTML pages and the OIDC response
+ */
+public class RFC252ServerMain {
```

Comment:
```suggestion
public class TestRFC252ServerMain {
```

We can go ahead and name this `Test` to call out that we are only using this for testing (like we did on the ODBC side, also the file name will have to change, of course)

Reply:
Sounds good, renamed it.